tests/config/test_step_config.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+from testfixtures import TempDirectory
+
+import os.path
+
+from tests.helpers.base_tssc_test_case import BaseTSSCTestCase
+
+from tssc.config import TSSCConfig, TSSCStepConfig, TSSCSubStepConfig
```

Comment:
is TSSCStepConfig, TSSCSubStepConfig needed?
import os.path should be placed above from testfixtures ...

Reply:
fixed